egs/wsj/s5/steps/libs/nnet3/xconfig/basic_layers.py

```diff
@@ -817,13 +817,6 @@ def get_full_config(self):
         output_dim = self.output_dim()
         transform_file = self.config['affine-transform-file']
 
-
```

Comment:
output-node for LDA should not always be generated inner FixedAffineLayer. we also generated when needed.